schemas/oic.wk.res-schema.json

```diff
@@ -8,24 +8,20 @@
       "properties": {
```

Comment:
This is about differentiated /oic/res response  per requesting Clients (OIC 1.1 vs OCF 1.0 Clients)
and already specified in below 
 
  11.3.5 Resource discovery using /oic/res & 
  11.3.6.4.1 /oic/res and retrieving of the resources